app/concerns/blockable.rb

```diff
@@ -13,6 +13,11 @@ def has_interaction_blocked?(user)
     user_ids_blocked_interaction.include?(user.id)
   end
 
+  def author_blocking?(post, author_ids)
+    return false unless post.authors_locked
+    author_ids.any?{ |author| self.blocking_post_users.include?(author) }
```

Comment:
I don't remember if I use this anywhere but if I do it should probably be 
 ```Block.where(blocking_user_id: author_ids, blocked_user: self).where("hide_me >= ?", Block::POSTS).exists?``` or similar

Reply:
I do not presently disagree but worth mentioning this flavor is more future-cache-friendly

Reply:
Hmm. Would intersection be equally friendly? I think it would work here?